### DISCUSSION TITLE
Persistent error 'Module not found error: megatron.fp16'

### DISCUSSION THREAD

**Discussion by tanmayg** [NVIDIA-NeMo/NeMo#3407] [Q&A]
I'm trying to fine tune the model: **BioMegatron345m-biovocab-50k-uncased** for a classfication task. 
I downloaded the pcorrespondingretrained weight file and the vocabulary from the NGC catalog. But the script has been continuously failing at the trainer.fit(model) step with the following error: ModuleNotFoundError: No module named 'megatron.fp16' I tried to find some solution but didn't get any leads. Would really appreciate fellow users help please.

**Comment by khcs** [comment]
We have new versions of the checkpoints that are uploaded on NGC in March.
Could you try with this: https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/biomegatron345m_biovocab_50k_uncased
